reports/aspace2alma/get_MARCxml_with_items.rb

```diff
@@ -7,23 +7,36 @@
 require_relative '../../helper_methods.rb'
 
 #configure sendoff to alma
-def alma_sftp (filename)
-  Net::SFTP.start(ENV['SFTP_HOST'], ENV['SFTP_USERNAME'], { password: ENV['SFTP_PASSWORD'] }) do |sftp|
-    sftp.upload!(filename, File.join('/alma/aspace/', File.basename(filename)))
-  end
-end
+# def send_to_alma_sftp (file_to_send)
```

Comment:
Did you mean to leave this commented out?

Reply:
This file is legacy code at this point. I'll remove it.